git-gui/po/README

```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
 
    When you have to change the order of words, you can add "<number>$"
    between '%' and the conversion ('s', 'd', etc.) to say "<number>-th
-   parameter to the format string is used at this point".  For example,
+   a parameter to the format string is used at this point".  For example,
```

Comment:
Did you mean to insert the "a" *before* the "<number>-th" (as opposed to appending it)?